charts/metallb/values.yaml

```diff
@@ -225,3 +225,15 @@ speaker:
     periodSeconds: 10
     successThreshold: 1
     timeoutSeconds: 1
+
```

Comment:
Any updates to `values.yaml` should have corresponding validations added in `values.schema.json` 

https://helm.sh/docs/topics/charts/#schema-files
https://www.arthurkoziel.com/validate-helm-chart-values-with-json-schemas/

Reply:
thanks